lib/tokens/jwt.go

```diff
@@ -81,6 +81,34 @@ func GenerateRefreshToken(secret []byte, expiryInSeconds int, u *models.User) (s
 
 	return t, nil
 }
+func ParseToken(secret []byte, token string) (int64, error) {
```

Comment:
I thought about re-using this logic, but I figured that for this PR it would be better to copy the function and not touch existing logic. We could always improve later.

Reply:
I was thinking the same. Why did you not re-use the existing code? 
checking the JWT and the claims seems essential to me, so I am worried future code changes might be missed because of the duplication. 

If it is really needed, maybe add a comment?